Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRulesCore"
 uuid = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
-version = "0.2.0"
+version = "0.3.0"
```

```suggestion
version = "0.3.0-DEV"
```

Lets no release this version, it can roll in with #30